docs/source/guide/pec-1-intro.myst

```diff
@@ -0,0 +1,139 @@
+frontend and backend. Here, for simplicity, we first convert the input circuit to the Mitiq internal
+representation (Cirq) and then simulate it.
+
+```{code-cell} ipython3
+from cirq import DensityMatrixSimulator, depolarize
+from mitiq.interface import convert_to_mitiq
+
+def executor(circuit, noise_level=0.01):
+    """Returns Tr[ρ |0⟩⟨0|] where ρ is the state prepared by the circuit
+    executed with depolarizing noise.
+    """
+    # Replace with code based on your frontend and backend.
+    mitiq_circuit, _ = convert_to_mitiq(circuit)
+    noisy_circuit = mitiq_circuit.with_noise(depolarize(p=noise_level))
+    rho = DensityMatrixSimulator().simulate(noisy_circuit).final_density_matrix
+    return rho[0, 0].real
+```
+
+The `executor` can be used to evaluate noisy (unmitigated) expectation values.
+
+```{code-cell} ipython3
+# Compute the expectation value of the |0><0| observable.
+noisy_value = executor(circuit)
+ideal_value = executor(circuit, noise_level=0.0)      
+print(f"Error without mitigation: {abs(ideal_value - noisy_value) :.5f}")
+```
+
+The aim of PEC is to reduce this error.
+
++++
+
+## Represent ideal gates as linear combinations of noisy gates
+
++++
+
+Before applying PEC, we need to represent the gates of the ideal `circuit` as linear combinations 
+of noisy gates. In Mitiq, one can express a gate representation via an `OperationRepresentation`
+object, which depends on the specific noise model of the backend.
+
++++
+
+In this example, we assume that local depolarizing noise of some fixed `noise_level` is acting after
+the application of each gate. For this simple noise model, one can easily obtain the necessary representations as follows:
+
+```{code-cell} ipython3
+from mitiq.pec.representations.depolarizing import represent_operations_in_circuit_with_local_depolarizing_noise
+
+noise_level = 0.01
+reps = represent_operations_in_circuit_with_local_depolarizing_noise(circuit, noise_level)
+print(f"{len(reps)} OperationRepresentation objects produced,")
+print(f"assuming depolarizing noise of strength p={noise_level}.")
+```
+
+As an example, we print the first `OperationRepresentation` in `reps`, showing how an ideal
+gate of `circuit` can be expressed as a linear combination of noisy gates:
+
+```{code-cell} ipython3
+print(reps[0])
+```
+
+**IMPORTANT:** For PEC to work properly, the noise model and the `noise_level`
+associated to the `executor` must be equal to those used when defining the
+`OperationRepresentation` objects.
+
++++
+
+## Apply PEC
+
++++
+Once the necessary `OperationRepresentation`s have been defined, probabilistic error cancellation can be applied as follows:
+
+```{code-cell} ipython3
+from mitiq import pec
+
+pec_value = pec.execute_with_pec(circuit, executor, representations=reps)
+
+print(f"Error without PEC: {abs(ideal_value - noisy_value) :.5f}")
+print(f"Error with PEC:    {abs(ideal_value - pec_value) :.5f}")
+```
```

Comment:
```suggestion
print(f"Error with PEC:    {abs(ideal_value - pec_value) :.5f}")
``` In this example, the noise has been reduced of a factor 30 with respect to the unmitigated case. 
```

Reply:
The results are non-deterministic so I rephrased it without giving a number for the error reduction factor.
See here: ba945aa